### PR TITLE
[Synapse] - Supports -Force parameter for Remove-* cmdlets

### PR DESCRIPTION
**Pull request by idear1203**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Synapse/Synapse/Properties/Resources.resx [hunk 1]
```diff
@@ -458,5 +458,40 @@ Are you sure you want to continue?</value>
   </data>
   <data name="RestoringSynapseSqlPool" xml:space="preserve">
     <value>Creating a new SQL pool restored from '{0}' in resource group '{1}' under parent Workspace '{2}' with name '{3}'.</value>
+  <data name="RemoveFirewallRule" xml:space="preserve">
```
Comment (isra-fel) [inline src/Synapse/Synapse/Properties/Resources.resx:461]:
```suggestion
  <data name="RestoringSynapseSqlPool" xml:space="preserve">
    <value>Creating a new SQL pool restored from '{0}' in resource group '{1}' under parent Workspace '{2}' with name '{3}'.</value>
  </data>
  <data name="RemoveFirewallRule" xml:space="preserve">
```
Looks like a bad merge

File: src/Synapse/Synapse/Commands/DataPlaneCommands/Artifact/Triggers/RemoveAzureSynapseTriggerSubscription.cs [hunk 2]
```diff
@@ -62,12 +65,21 @@ public override void ExecuteCmdlet()
 
             if (this.ShouldProcess(this.WorkspaceName, String.Format(Resources.RemovingSynapseTriggerSubscription, this.Name)))
             {
-                SynapseAnalyticsClient.StartUnsubscribeTriggerFromEvents(this.Name);
-                if (PassThru)
-                {
-                    WriteObject(true);
-                }
             }
```
Comment (wonner) [inline src/Synapse/Synapse/Commands/DataPlaneCommands/Artifact/Triggers/RemoveAzureSynapseTriggerSubscription.cs:68]: Also remove these lines?

Comment (idear1203) [reply src/Synapse/Synapse/Commands/DataPlaneCommands/Artifact/Triggers/RemoveAzureSynapseTriggerSubscription.cs:68]: Good catch!

File: src/Synapse/Synapse/Properties/Resources.resx (hunk 1 above)
Comment (idear1203) [reply src/Synapse/Synapse/Properties/Resources.resx:461]: Thanks. Will fix in the coming commit

**Review by isra-fel — APPROVED** [review]
LGTM

**Pull request merged**